conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
By "test conda build" you mean "run the full unit tests suite, not just
check a few imports"? Even on Travis where those tests were just run?
On Thu, Nov 5, 2015 at 8:58 AM Eric Dill <notifications@github.com> wrote:

> In conda-recipe/run_test.sh
> <https://github.com/NSLS-II/metadatastore/pull/184#discussion_r44013264>:
>
> > @@ -0,0 +1 @@
> > +nosetests -v metadatastore --cover-package metadatastore --with-coverage
>
> Oh this is what is was:
>
> We will test the conda build on Travis. We will test the conda build in
> the git repo. We will not include the test script with the recipes on
> conda-prescriptions. The logic being that our tagged versions have been
> thoroughly tested, but if you are building a conda package from a non
> tagged version it is better to test it during the build.
>
> Also it is very easy to skip the testing by simply deleting the
> run_tests.sh file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/metadatastore/pull/184/files#r44013264>.
>